poc/components/PocButton.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Text, View, TouchableHighlight } from 'react-native';
+import styles from '../styles';
+
+export default function PocButton(props) {
+    return (
+        <View>
+            {/* TODO: invoke function passed in on state */}
```

Comment:
What's the follow-up work here? Looks like you are already passing in the event handler on the props, and it seemed to work when I tested it locally.

Reply:
Whoops need to remove